cineast-api/src/main/java/org/vitrivr/cineast/api/messages/query/TemporalQuery.java

```diff
@@ -76,4 +77,8 @@ public TemporalQueryConfig getTemporalQueryConfig() {
   public MessageType getMessageType() {
     return MessageType.Q_TEMPORAL;
   }
+
+  public void setTimeDistances(List<Float> distances) {
```

Comment:
If you have merged vitrivr-ng correctly, this field is not required. It is present in the `TemporalQueryConfig` (it's possible that this is my bad). If you open a PR for the vitrivr-ng branch, I can check that.

Reply:
I didn't touch vitrivr-ng at all, just used the branch that has the pose query interface. Without this setter, it didn't work. That branch might not be completely current though, I'll have to check.

Reply:
With the most recent version, it's no longer needed, so I removed it.